stonesoup/models/transition/linear.py

```diff
@@ -307,12 +308,92 @@ def covar(self, time_interval, **kwargs):
         return CovarianceMatrix(covar)
 
 
-class ConstantVelocity(LinearGaussianTransitionModel, TimeVariantModel):
+class ConstantNthDerivative(LinearGaussianTransitionModel, TimeVariantModel):
+    r"""Model based on the Nth derivative of position being constant,
```

Comment:
May be worth using different terminology to  "position". I suspect that this can also be applied to variables like bearing, orientation, etc.